### PR TITLE
[TNW-777] Integrate usage of nonce tokens

### PR DESCRIPTION
**Pull request by wojciech-gruszka-tg**

### CODE REVIEW

**Review by grantas-valiukenas-tg — APPROVED** [review]

File: src/components/OidcClient.js [hunk 1]
```diff
@@ -25,6 +25,11 @@ class OidcClient {
         queryParams.append('code_challenge', codeChallenge)
         queryParams.append('redirect_uri', config.redirectUri)
 
+        const windowQueryParameter = new URLSearchParams(window.location.search);
+        if (windowQueryParameter.has('acr_values')) {
+            queryParams.append('acr_values', windowQueryParameter.get('acr_values'))
```
Comment (grantas-valiukenas-tg) [inline src/components/OidcClient.js:30]: shouldn't you add `acr` parameter as well?

Comment (wojciech-gruszka-tg) [reply src/components/OidcClient.js:30]: Why should I? This solution works well

Comment (grantas-valiukenas-tg) [reply src/components/OidcClient.js:30]: I was looking at curity-server code and in one place it expects `acr` query paramter with some value. That's why I thought it should be added manually.

Comment (wojciech-gruszka-tg) [reply src/components/OidcClient.js:30]: `acr_values` parameter is passed to the Curity server at the beginning of the authentication flow. Then curity server sends the client by using `redirect-response` to call the Curity server with the `acr` parameter.

**Pull request merged**